$OpenBSD: patch-tmac_an-old_tmac,v 1.6 2017/07/04 17:44:13 schwarze Exp $
Set default operating system name (local change).
Tweak page headers (local change).
Index: tmac/an-old.tmac
--- tmac/an-old.tmac.orig
+++ tmac/an-old.tmac
@@ -147,14 +147,14 @@
 .    ds an-section "\\$2\"
 .    ds an-extra1 "\\$3\"
 .    ie (\\n[.$] > 3)    .ds an-extra2 "\\$4\"
-.    el                  .ds an-extra2 \"Sun Release 4.0
+.    el                  .ds an-extra2 OpenBSD
 .
 .    ie (\\n[.$] > 4)    .ds an-extra3 "\\$5\"
 .    el \{ .ie '\\$2'1'  .ds an-extra3 General Commands Manual
 .    el \{ .ie '\\$2'2'  .ds an-extra3 System Calls Manual
 .    el \{ .ie '\\$2'3'  .ds an-extra3 Library Functions Manual
-.    el \{ .ie '\\$2'3p' .ds an-extra3 Perl Programmers Reference Guide
-.    el \{ .ie '\\$2'4'  .ds an-extra3 Kernel Interfaces Manual
+.    el \{ .ie '\\$2'3p' .ds an-extra3 Perl Library Manual
+.    el \{ .ie '\\$2'4'  .ds an-extra3 Device Drivers Manual
 .    el \{ .ie '\\$2'5'  .ds an-extra3 File Formats Manual
 .    el \{ .ie '\\$2'6'  .ds an-extra3 Games Manual
 .    el \{ .ie '\\$2'7'  .ds an-extra3 Miscellaneous Information Manual
